test/elixir/test/bulk_docs_test.exs

```diff
@@ -152,4 +163,11 @@ defmodule BulkDocsTest do
     assert resp.body["error"] == "bad_request"
     assert resp.body["reason"] == reason
   end
+
+  defp random_string(length) do
+    raw = :crypto.strong_rand_bytes(length)
```

Comment:
I didn't like this initially. However, unfortunately this is the easiest way to generate random bytes in Elixir. The alternative is probably slower:
 
```
Enum.map(0..length, fn _ -> :rand.uniform(255) end) 
  |> List.to_string 
  |> Base.url_encode64
  |> binary_part(0, length)
```

Reply:
So should we leave it as-is?

Reply:
Leave it as is.